app/assets/javascripts/discourse/app/routes/app-route-map.js

```diff
@@ -141,17 +141,23 @@ export default function () {
         "userPrivateMessages",
         { path: "/messages", resetNamespace: true },
         function () {
-          this.route("new");
-          this.route("unread");
-          this.route("archive");
-          this.route("sent");
-          this.route("warnings");
-          this.route("group", { path: "group/:name" });
-          this.route("groupArchive", { path: "group/:name/archive" });
-          this.route("groupNew", { path: "group/:name/new" });
-          this.route("groupUnread", { path: "group/:name/unread" });
-          this.route("tags");
-          this.route("tagsShow", { path: "tags/:id" });
```

Comment:
In an ideal world, we would deprecate the old routes but I have done a quick search through all known plugins and no plugins are currently relying on those routes. There is a chance we could break plugins here but I'll like to see some smoke first before committing to the effort of deprecating client side routes.